src/quantum/subbenchmarks.jsx

```diff
@@ -38,6 +41,26 @@ const Graph = ({ data, name, url }) => (
         />
       </a>
     </div>
+    <ChartJsWrapper
+      title={name}
+      type='line'
+      height={80}
+      options={generateOptions()}
+      data={{
+        datasets: data.map((part) => {
+          return {
+            data: part.map(({ datetime, value }) => {
+              return {
+                x: datetime,
+                y: value,
+              };
+            }),
+            label: name,
+          };
+        }),
+      }}
+    />
+
     <MetricsGraphics
```

Comment:
Remove `MetricsGraphics`

Reply:
yes, I will remove metrics graphics when the other is working; I just need something to show to prove it I did not break other stuff